packages/stepper/src/constants/i18n.ts

```diff
@@ -0,0 +1,5 @@
+const STEPPER_NAMESPACE = 'stepper-app';
```

Comment:
Just for your information, we have to setup some stuff around i18n while adding a new namespace for a package 
It will need :
- New XTM package definition
- I18n extract configuration
- Script in package.json to ensure the extraction is done